src/gbd_mapping/coverage_gap.py

```diff
@@ -29,42 +30,93 @@
         affected_causes=(causes.measles, ),
         affected_risk_factors=(),
     ),
-    lack_of_immediate_assessment_and_stimulation=CoverageGap(
-        name='lack_of_immediate_assessment_and_stimulation',
+    tmp_lack_of_lipid_lowering_therapy=CoverageGap(
```

Comment:
What's this tmp coverage gap and do we actually want it in the mapping if it's temporary?

Reply:
oh gosh.. `gbd_mapping` is built automatically based on the file in J drive, meaning that someone put the tmp file. Do you know about this file? if it's not something leaving there, we need to delete it.

Reply:
It's not a folder I've seen before. @halfhorst is this something you're using for testing replace?